include/abb_librws/rws_interface.h

```diff
@@ -512,6 +512,16 @@ class RWSInterface
    */
   bool endSubscription();
 
+  /**
+   * \brief Close the active subscription connection.
+   *
+   * This will cause waitForSubscriptionEvent() to return or throw.
+   * It does not delete the subscription from the controller.
+   *
```

Comment:
```suggestion
   * Preferred way to close the subscription is to request the robot controller to end it via endSubscription().
   *
```
Just as a note that it is "nicer" for the robot controller to use endSubscription() when possible.